libfwupdplugin/fu-device.c

```diff
@@ -7661,6 +7663,8 @@ fu_device_add_json(FwupdCodec *codec, JsonBuilder *builder, FwupdCodecFlags flag
 		json_builder_add_string_value(builder, str);
 	}
 
+	fwupd_dev_interface->add_json(codec, builder, flags);
+
 	/* subclassed */
 	if (device_class->add_json != NULL)
```

Comment:
Shall I "else" this and only call the parent "add_json" in this case? But then all the "common" properties would not appear